src/components/shared/bylineCard.tsx

```diff
@@ -0,0 +1,200 @@
+import { remSpace } from '@guardian/src-foundations';
+import { withDefault, map, fromNullable } from '@guardian/types/option';
+import { pipe2 } from 'lib';
+import { neutral, opinion, text } from '@guardian/src-foundations/palette';
+import { Design, Display, Format } from '@guardian/types/Format';
+import { darkModeCss } from 'styles';
+import { getPillarStyles, pillarFromString } from 'pillarStyles';
+import { SvgQuote } from '@guardian/src-icons';
+import { makeRelativeDate } from 'date';
+
+interface Props {
+    relatedItem: RelatedItem;
+}
+
+const borderColor = (format: Format): SerializedStyles => {
+    return css`1px solid ${getPillarStyles(format.pillar).kicker}`
+}
+
+const listStyles = (format: Format): SerializedStyles => {
+    return css`
+        background: white;
+        margin-right: ${remSpace[3]};
+        flex: 0 0 15rem;
+        height: 100%;
+        display: flex;
+        flex-direction: column;
+        justify-content: space-between;
+        border-top : ${borderColor(format)};
+
+        &.fade {
+            opacity: .7;
+        }
+
+        ${darkModeCss`
+            background: ${neutral[7]};
+        `}
+    `;
+}
+
+const bylineImage = css`
+    border-radius: 50%;
+    right: 0.625rem;
+    top: 0.375rem;
+    overflow: hidden;
+    height: 8.25rem;
+    width: 8.25rem;
+    contain: paint;
+    background-color: ${opinion[400]};
+    float: right;
+    margin: 0 ${remSpace[2]} 0 0;
+
+    img {
+        margin: auto;
+        position: absolute;
+        top: 0;
+        bottom: 0;
+        right: 0;
+        height: 8.25rem;
+        left: -0.625rem;
+    }
+`;
+
+const anchorStyles = css`
+    color: ${neutral[7]};
+    text-decoration: none;
+    ${darkModeCss`
+        color: ${neutral[86]};
+    `}
+`;
+
+const headingWrapperStyles = css`
+    padding: ${remSpace[2]};
+    min-height: 150px;
+`
+
+const headingStyles: SerializedStyles = css`
+        ${headline.xxsmall()}
+        margin: 0 0 ${remSpace[2]} 0;
+    `;
+
+const cardStyles: SerializedStyles = css`
+        background-color : ${opinion[800]};
+        ${headline.xxsmall()}
+    `;
+
+
+const commentIconStyle = (): SerializedStyles => {
+    return css`
+        width: 2.0rem;
+        height: 1.4375rem;
+        display: inline-block;
+        fill: ${opinion[400]};
+        vertical-align: text-top;
+        margin-top: -3px;
+        margin-right: -2px;
+    `;
+}
+
+const bylineStyles: SerializedStyles = css`
+    color: ${opinion[400]};
+`;
+
+const byline = (relatedItem: RelatedItem): ReactElement | null => {
+    return pipe2(
+        fromNullable(relatedItem.byline),
+        map(byline => {
+            return <div css={bylineStyles}>{byline}</div>
+        }),
+        withDefault<ReactElement | null>(null)
+    )
+}
+
+const cardImage = (relatedItem: RelatedItem): ReactElement | null =>
+    <div css={bylineImage}>
+        <img alt={relatedItem.byline ?? "Byline image"} src={relatedItem?.bylineImage}/>
```

Comment:
Does this mean we'll get some images without a `src` if `bylineImage` is `undefined`?

Reply:
the bylineCard would only render if the `bylineImage` was present, but we're using the `RelatedItem` type for the parameter. I've added an undefined check.